dds/DCPS/RTPS/ICE/EndpointManager.cpp

```diff
@@ -723,14 +723,12 @@ void EndpointManager::compute_active_foundations(ActiveFoundationSet& a_active_f
 void EndpointManager::check_invariants() const
 {
   // Check for expired deferred checks.
-  size_t deferred = 0;
   for (DeferredTriggeredChecksType::const_iterator pos = deferred_triggered_checks_.begin(),
          limit = deferred_triggered_checks_.end(); pos != limit; ++pos) {
     const DeferredTriggeredCheckListType& list = pos->second;
 
     OPENDDS_ASSERT(!list.empty());
     OPENDDS_ASSERT(list.front().expiration_date >= MonotonicTimePoint::now() - agent_impl->get_configuration().server_reflexive_address_period() - agent_impl->get_configuration().server_reflexive_address_period());
-    deferred += list.size();
```

Comment:
Now `list` is also an unused variable.  I wonder if/when compilers will start warning about loops with empty bodies.